scripts/DolbyIOManager.cs

```diff
@@ -40,30 +40,34 @@ public class DolbyIOManager : MonoBehaviour
         /// <returns>An asynchronous task containing the token.</returns>
         public static async Task<string> GetToken(string key, string secret)
         {
-            string result = "";
-
-            using (var client = new HttpClient())
+            return await Task.Run(async () =>
             {
-                var request = new HttpRequestMessage(HttpMethod.Post, " https://session.voxeet.com/v1/oauth2/token");
-                var auth = $"{Uri.EscapeUriString(key)}:{Uri.EscapeUriString(secret)}";
-                
-                request.Headers.Authorization = new AuthenticationHeaderValue("Basic", $"{Convert.ToBase64String(Encoding.UTF8.GetBytes(auth))}");
-                request.Content = new FormUrlEncodedContent(new Dictionary<string, string> {{"grant_type", "client_credentials"}});
+                string result = "";
+
+                using (var client = new HttpClient())
```

Comment:
It is recommended to keep a static variable for the `HtpClient` to not create a new instance every time you need to make an HTTP call. It opens a socket connection every time and it takes time to close it.